src/main/java/com/epam/tc/hw9/service/TrelloServiceObject.java

```diff
@@ -0,0 +1,115 @@
+package com.epam.tc.hw9.service;
+
+import static org.hamcrest.Matchers.lessThan;
+
+import com.epam.tc.hw9.dto.BoardDto;
+import com.epam.tc.hw9.dto.CardDto;
+import com.epam.tc.hw9.dto.ListDto;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.http.HttpStatus;
+
+public class TrelloServiceObject {
+
+    private final Method requestMethod;
+    private final Map<String, String> parameters;
+
+    private TrelloServiceObject(Map<String, String> parameters, Method method) {
+        this.parameters = parameters;
+        this.requestMethod = method;
+    }
+
+    public static ApiRequestBuilder requestBuilder() {
+        return new ApiRequestBuilder();
+    }
+
+    public static class ApiRequestBuilder {
+
+        private final Map<String, String> parameters = new HashMap<>();
+        private Method requestMethod = Method.GET;
```

Comment:
why GET only?

Reply:
I use this method by default and for example don't call method 'setMethod' when create getABoardTest

Reply:
here u should answer for yourself why default method is GET and not POST.

Better is not to use default method at all. just use directly supplied